Refrigerator/Enums.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
```

Comment:
commom use of access modifiers a long your object is necessary.
please over view your project and mention the relevant: public, private, protected.